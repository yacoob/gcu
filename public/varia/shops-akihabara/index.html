<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/c/2013-11-12/s1920/IMG_20131110_162150.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/c/2013-11-12/s1920/IMG_20131110_162150.jpg'/>
  <meta name='description' content='tactical grace presents: Shops of Akihabara'/>
  <link href='http://gcu.tactical-grace.net/varia/shops-akihabara/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>Shops of Akihabara | tactical grace</title>
  <meta content='Shops of Akihabara | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/varia/shops-akihabara/'>Shops of Akihabara</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a href='/varia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a href='/varia/sd-kshatriya/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2013-11-12'>2013-11-12</h2>
  </div>
  <div class='row voffset-small'>
  <div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-11-12/s1920/IMG_20131110_162150.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-11-12/s400-p/IMG_20131110_162150.jpg' title='&amp;quot;No, *I* am Zogok!&amp;quot;' alt='&amp;quot;No, *I* am Zogok!&amp;quot;' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-11-12/s1920/IMG_20131110_162156.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-11-12/s400-p/IMG_20131110_162156.jpg' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-11-12/s1920/IMG_20131111_144020.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-11-12/s400-p/IMG_20131111_144020.jpg' title='Featuring Maid Zogok.' alt='Featuring Maid Zogok.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-11-12/s1920/IMG_20131111_110016.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-11-12/s400-p/IMG_20131111_110016.jpg' title='Responding with lethal force.' alt='Responding with lethal force.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-11-12/s1920/1384772930447.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-11-12/s400-p/1384772930447.jpg' title='Century Color! Million Color!' alt='Century Color! Million Color!' />
    </a>
  </div>
</div>
  </div>
  <div class='row voffset-top'>
    <h2 id='2015-05-14'>2015-05-14</h2>
  </div>
  <div class='row voffset-small'>
  <div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-05-12/s1920/IMG_20150410_150335.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-05-12/s400-p/IMG_20150410_150335.jpg' title='TSUKAME PURAIDO!' alt='TSUKAME PURAIDO!' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-05-12/s1920/IMG_20150410_151530.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-05-12/s400-p/IMG_20150410_151530.jpg' title='Grimoires and High Mocks extravaganza.' alt='Grimoires and High Mocks extravaganza.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-05-12/s1920/IMG_20150410_151548.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-05-12/s400-p/IMG_20150410_151548.jpg' title='So many great ideas!' alt='So many great ideas!' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-05-12/s1920/IMG_20150411_112128.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-05-12/s400-p/IMG_20150411_112128.jpg' title='Ultimate nipper was sold out... but its mascot was in. A nipper with an official mascot.' alt='Ultimate nipper was sold out... but its mascot was in. A nipper with an official mascot.' />
    </a>
  </div>
</div>
  </div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link href='/css/jquery.fancybox.css' rel='stylesheet'>
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='/js/jquery.fancybox.min.js'></script>
<script src='/js/jquery.fancybox-media.js'></script>
<script src='/js/screenfull.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>