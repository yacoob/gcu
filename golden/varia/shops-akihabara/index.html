<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00265.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00265.jpg'/>
  <meta name='description' content='tactical grace presents: Shops of Akihabara'/>
  <link href='https://gcu.tactical-grace.net/varia/shops-akihabara/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>Varia: Shops of Akihabara | tactical grace</title>
  <meta content='Varia: Shops of Akihabara | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/varia/shops-akihabara/'>Varia: Shops of Akihabara</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/varia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/varia/sd-kshatriya/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a data-toggle='tooltip' title='next kit' href='/varia/wonfes-winter-2017/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2013-11-12'>2013-11-12</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00265.jpg' data-sub-html='&amp;quot;No, *I* am Zogok!&amp;quot;'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00265.jpg' alt='&amp;quot;No, *I* am Zogok!&amp;quot;' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00266.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00266.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00268.jpg' data-sub-html='Featuring Maid Zogok.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00268.jpg' alt='Featuring Maid Zogok.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00267.jpg' data-sub-html='Responding with lethal force.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00267.jpg' alt='Responding with lethal force.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00264.jpg' data-sub-html='Century Color! Million Color!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00264.jpg' alt='Century Color! Million Color!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
  <div class='row voffset-top'>
    <h2 id='2015-05-14'>2015-05-14</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00652.jpg' data-sub-html='TSUKAME PURAIDO!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00652.jpg' alt='TSUKAME PURAIDO!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00653.jpg' data-sub-html='Grimoires and High Mocks extravaganza.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00653.jpg' alt='Grimoires and High Mocks extravaganza.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00654.jpg' data-sub-html='So many great ideas!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00654.jpg' alt='So many great ideas!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00655.jpg' data-sub-html='Ultimate nipper was sold out... but its mascot was in. A nipper with an official mascot.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00655.jpg' alt='Ultimate nipper was sold out... but its mascot was in. A nipper with an official mascot.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>